src/app/chat/chat.service.ts

```diff
@@ -178,4 +186,203 @@ export class ChatService {
   private getRamdomNumber() {
```

Comment:
We need a way to make sure two users doesn't use the same color. This should be inside `getRandomColor()`

Reply:
The logic is selecting random color two users can have same color to identify those two that's why we using Avatar text. This logic similar to android contact app. This is the way we use in app v1 and this is the way designers also agreed. we have 19 colors that designers selected we are using one random color from them.